public/app/features/search/page/components/SearchResultsTable.tsx

```diff
@@ -141,14 +144,42 @@ export const SearchResultsTable = React.memo(
         return (
           <div key={key} {...rowProps} className={className}>
             {row.cells.map((cell: Cell, index: number) => {
+              const href = onClickItem ? url : undefined;
+
+              let userProps: { href?: string; onClick?: (event: React.MouseEvent<HTMLElement>) => void } = {
+                href,
+                onClick: onClickItem,
+              };
+
+              if (cell.column.id === 'column-name' && href) {
+                const parent = response.view.dataFrame.fields[5].values[rowIndex];
+                const kind = response.view.dataFrame.fields[0].values[rowIndex];
+                const existingOnClick = onClickItem;
```

Comment:
```suggestion
                const item = response.view.get(rowIndex);
                const parent = item.location || 'general';
                const parentType = parent === 'general' ? 'general' : 'folder';
```
I think we can do this instead of using hardcoded indices.